lib/src/server.dart

```diff
@@ -56,25 +56,54 @@ abstract class Service {
 /// Listens for incoming gRPC calls, dispatching them to a [ServerHandler].
 class Server {
   final Map<String, Service> _services = {};
-  ServerSocket _server;
+  final int port;
+  final SecurityContext _securityContext;
 
+  ServerSocket _insecureServer;
+  SecureServerSocket _secureServer;
```

Comment:
Not entirely clear why we should need both. Don't the two types have a common ancestor with a `close` method?